applepay/utils.py

```diff
@@ -1,20 +1,38 @@
+"""
+The utils in this module are used to verify and parse a decoded
```

Comment:
Sure, how about something like this?

"The utilities assume the token has already be base64 decoded and dumped from json into a
python dictionary."